### ISSUE TITLE
Appending does not remotely create file

### ISSUE THREAD

**Issue by JPLanh** [JPLanh/DistributedFileSystem#3]
Appending function does not create pages on the remote chord, as the current code has it that it will create the appropriate page on the local computer with the guid ID, but we need to have it remote onto the other computer and have the page created in the other computer in the guid id folder.

**Comment by JPLanh** [comment]
Function has been implemented, now just needed to be test.

**Comment by JPLanh** [comment]
After few changes, I should create some sort of progress bar while uploading to show any progress instead of nothing at all. Also i've encountered an error in Client.py in line 92, where it says unsupported operand type(s) for +: 'int' and 'str'

**Comment by JPLanh** [comment]
Turns out, when i was having my method to append files it was using the guid in integer values to try and open a file. python doens't like it when trying to open a file has a mix of integer and string.

**Comment by JPLanh** [comment]
new changes has allowed the remote workstation to create the respective page on the workstation, but it seem like all pages are going to the remote workstation instead of splitting among the two workstation. It doesn't help that the two chord's guid are relatively close to each other while testing (214323770453679595168069373855162065930, and the other was like 23...). However the guid with pages < 214... and pages pages > 214.... are still going into the folder and none are showing up on the 23... workstation.

**Comment by JPLanh** [comment]
this issue caused a cluster of issue, so the sub issue has now been resolved which allow this issue to be resolved as well. According to #15, allow the remote creation of files, but due to how the chord are being remotely access, it ran into the issue #16. The issue revolving pages going to only one station has been resolved as well. Even though the two chord's guid are relatively close to one and another I got lucky in the creation of the pages that one of the page lies in between the two chord and the page was created in the respective server.
